subprojects/resources-http/src/test/groovy/org/gradle/internal/resource/transport/http/HttpClientConfigurerTest.groovy

```diff
@@ -137,12 +139,12 @@ class HttpClientConfigurerTest extends Specification {
         configurer.configure(httpClientBuilder)
 
         then:
-        def basicCredentials = httpClientBuilder.credentialsProvider.getCredentials(new AuthScope(AuthScope.ANY_HOST, AuthScope.ANY_PORT))
+        def basicCredentials = httpClientBuilder.credentialsProvider.getCredentials(new AuthScope("host", 1234))
```

Comment:
You've got these "magic strings/numbers" hard coded in multiple places in the test, do you want to extract them to a variable or constants in the scope of the test or class.